lib/hammer_cli_foreman/user.rb

```diff
@@ -43,6 +43,7 @@ def extend_data(user)
         user["locale"] ||= _('default') if user.has_key?("locale")
         user["timezone"] ||= _('default') if user.has_key?("timezone")
         user["full_name"] = [user["firstname"], user["lastname"]].join(' ')
+        user["cached_usergroups"] = user["cached_usergroups"] - user["usergroups"]
```

Comment:
This gave me `Error: undefined method `-' for nil:NilClass` when I tried to run the command without the server counterpart. Could you please use `user.fetch('cached_usergroups', [])` to keep compatibility with older versions of Foreman?